BUILD.gn

```diff
@@ -368,6 +369,8 @@ source_set("electron_lib") {
     "//third_party/webrtc_overrides:webrtc_component",
     "//third_party/widevine/cdm:headers",
     "//third_party/zlib/google:zip",
+    "//ui/accessibility",
+    "//ui/base",
```

Comment:
We probably don't want to take on these new dependencies wholesale, as we should just pull out what we need here for a change of this relatively small size

Reply:
Builds fine for me, pushed a fixup.